### ISSUE TITLE
A11y: Generic avatars should have an accessible color contrast ratio

### ISSUE THREAD

**Issue by capeoples** [Esri/calcite-components#1321]
### Summary
In an accessibility audit performed by Level Access, it was flagged that the contrast ratio of the generic profile thumbnails does not pass accessibility standards. While most of the time these are decorative images, we could darken the text to a color like #6E6E6E to make this pass AA standards and still look good. 

### Actual Behavior
<img width="146" alt="Screen Shot 2020-11-25 at 4 35 49 PM" src="https://user-images.githubusercontent.com/30473086/100294661-4eb59b80-2f3c-11eb-8805-81df01c5849d.png">

### Expected Behavior
<img width="134" alt="Screen Shot 2020-11-25 at 4 34 06 PM" src="https://user-images.githubusercontent.com/30473086/100294668-52492280-2f3c-11eb-8bd1-6fc543bea66c.png">

**Comment by julio8a** [comment]
I think @paulcpederson had mentioned that on the current Online avatar, they can infinitely generate color parings. I'm wondering if we can generate a set?

The thing would be to avoid people having the same color?

CCing @bstifle (The color guru2.0)

**Comment by bstifle** [comment]
yeah we worked on these so that they would always pass contrast. text color looks off. should be 6a6a6a.

@paulcpederson hel!p

**Comment by paulcpederson** [comment]
It... is #6a6a6a , no help needed my dude.

@capeoples the offending avatar is probably not a calcite-avatar yet? In that case the answer your a11y woes is to update it to use `calcite-avatar`!

**Comment by paulcpederson** [comment]
Feel free to reopen if this is indeed a bug in CC

**Comment by julio8a** [comment]
Thanks all!
